mmdet/models/necks/ssh.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Tuple
+
+import torch
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+from mmengine.model import BaseModule
+
+from mmdet.registry import MODELS
+from mmdet.utils import ConfigType, OptConfigType, OptMultiConfig
+
+
+class SSHModule(BaseModule):
```

Comment:
SSHDetModule

Reply:
decompose the SSHModule into SSHContextModule and SSHDetModule